umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif10/spatial.clj

```diff
@@ -53,8 +57,9 @@
      :HorizontalSpatialDomain      (let [[geom] (select spatial "Geometry")]
                                      {:Geometry (parse-geometry geom)
                                       :ZoneIdentifier (value-of spatial "Zone_Identifier")})
-     :VerticalSpatialDomains       (for [vert-elem (select spatial "Vertical_Spatial_Info")]
-                                     (fields-from vert-elem :Type :Value))
+     :VerticalSpatialDomains       (->> (select spatial "Vertical_Spatial_Info")
+                                        spatial-conversion/convert-vertical-spatial-domains-from-xml
+                                        (map #(update % :Type (fn [x] (string/replace x "_" " ")))))
```

Comment:
I think what's being done in the map should be done in `spatial-conversion/convert-vertical-spatial-domains-from-xml`. That function should squish them if needed.